src/database/sql/audio.py

```diff
@@ -478,7 +513,8 @@ def get_loudness_integrated(self, audio_id: str) -> str:
         query = """
             SELECT Loudness_integrated, last_updated
             FROM Audio
-            WHERE audio_release=:rel AND audio_side=:side AND audio_track=:track
+            WHERE audio_release=:rel AND audio_side=:side
+            AND audio_track=:track
```

Comment:
If you want this AND `audio_track=:track` to be on a new line, please indent it.